docs/changelog.md

```diff
@@ -122,6 +125,17 @@ Becomes
 ```
 
 
+### Contact details organism
```

Comment:
Hoe komt dit in deze PR?

Reply:
Is in orde. Dit is de gebuilde changelog die er bij mijn commit nog niet inzat.

Reply:
I know 😛 Graag gedaan